FirebaseAdmin/FirebaseAdmin/Auth/GetAccountInfoRequest.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirebaseAdmin.Auth
+{
+    /// <summary>
+    /// Represents a request to lookup account information.
+    /// </summary>
+    internal sealed class GetAccountInfoRequest
+    {
+        private IList<string> uids = null;
```

Comment:
If you want you can preinitialize these to empty lists:

```
private readonly IList<string> uids = new List<string>();
```

Then check `!uids.Any()` in your Build methods.